.gitignore

```diff
@@ -11,4 +11,8 @@ pom.xml.asc
 .hgignore
 .hg/
 .cpcache/
-.clj-kondo/.cache
+.clj-kondo/
```

Comment:
I need to think more how I intend to handle .gitignore and external contributors. Mind removing those changes?

Reply:
Sure thing